### PR TITLE
CRAF-16: Created SQL + Liquibase for Customer table

### PR DESCRIPTION
**Pull request by Luke-Bradford-BBD**

### CODE REVIEW

File: changelog.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 databaseChangeLog:
-   - include:
-       file: sql/test.sql
\ No newline at end of file
+    - include:
+          file: sql/test.sql
```
Comment (ryanbasiltrickett) [inline changelog.yaml]: You can remove this file

File: sql/customer.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+-- liquibase formatted sql
+
+-- changeset lbradford:table-customer-creation
+CREATE TABLE "Customer" (
```
Comment (ryanbasiltrickett) [inline sql/customer.sql:4]: All should be NOT NULL, instead of NON NULL

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]
Please do requested changes

File: sql/customer.sql (hunk 2 above)
Comment (Luke-Bradford-BBD) [reply sql/customer.sql:4]: My mistake, correcting it now

**Review by LucindaBotes — APPROVED** [review]

**Review by ryanbasiltrickett — APPROVED** [review]

**Pull request merged**